index.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+    </style>
+</head>
+<body>
+
+    <header>
+        <div class="nav-menu-content">
+
+            <div class="logo"><a class=mylogo> My Logo </a></div>
+
+            <nav>
+                <ul class="nav-menu">
+                    <li class="nav-items"><a href="#">Portfolio</a></li>
+                    <li class="nav-items"><a href="#">About</a></li>
+                    <li class="nav-items"><a href="#">Contact</a></li>
+                  </ul>
+
+                    <div class="hamburger">
+
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
+            </nav>
+        </div>
+
+    </header>
+    <section class="homepage">
+        <div class="introduction">
+            <div class="intro"><h1>I'm Tasnimul <br>Glad to meet you!</h1></div>
+            <div class="me">
+                <p>
+                    I am a software developer. I have done some interesting projects using
+                    react js, node js and ruby on rails. I am learning new things and
+                    implementing them instantly. I can help you build a project, feature
+                    or website look through some of work and experience if you like what
+                    you see and have a project you need coded, don't hesitate to contact
+                    me.
+                </p>
+            </div>
+            <div class="connect">
+                <h3><a> Let's connect </a> </h3>
+            </div>
+            <div class="social">
+                <ul>
+                    <li class="icon"><a href=""><img src="Image/twitter.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/github.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/medium.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/angellist.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/Linkedinicon.png" alt="" /></a></li>
+                </ul>
+
+            </div>
+        </div>
+    </section>
+
+    <section class="portfolio-1">
+        <div class="project-1">
```

Comment:
As the last reviewer mentioned, it's key to this task that we try to copy as much as possible Figma's design. 

Please address the following issues:
- [ ] Position of elements - Elements(mostly text elements) in the card should follow the positioning shown in Figmas design
- [ ] Cards are overflowing each other
- [ ] Resize image of cards 

Yours
![image](https://user-images.githubusercontent.com/15807343/154713107-548142af-50f8-4d32-9b96-ae4f22f59374.png)

Figma's
![](https://user-images.githubusercontent.com/70483835/154630820-dcaa99b3-c104-4f0a-9e0a-f42f571e25c0.png)

![image](https://user-images.githubusercontent.com/15807343/154714502-a15a58a8-bc65-46a6-b432-71c4a96dc154.png)

![image](https://user-images.githubusercontent.com/15807343/154714789-1fa79b3b-3b7c-4655-8933-6273fc10a150.png)

![image](https://user-images.githubusercontent.com/15807343/154714368-f91c1a74-90c4-435b-a987-da4c3c05c6e6.png)

![image](https://user-images.githubusercontent.com/15807343/154714679-cd067a2f-e237-483a-90ed-8eaef653e224.png)